ad9959/src/lib.rs

```diff
@@ -582,7 +582,9 @@ impl ProfileSerializer {
         }
 
         if let Some(acr) = acr {
-            self.add_write(Register::ACR, &acr.to_be_bytes());
+            let mut data = [0; 3];
+            data[1..=2].copy_from_slice(&acr.to_be_bytes());
```

Comment:
Better have `acr: Option<u32>` and copy the 3 low bytes.
We should cover the entire register (among other things, `Amplitude multiplier enable` to be set as well).

Reply:
The upper 8 bytes is actually just the ramp control register, which we don't really support at all right now, so that's why I was opting to default them to zero. The enable bit is bit 4 of ACR[1], which fits in the lower 16 bits

Reply:
If we take the full ACR the full ramp functionality is supported and there is no mismatch between the meaning of `Acr` and the actual register content. Why restrict it to the 16 LSBs? That still exposes bits (15, 14, 11, 10) which then can/must not be used because of that restriction.